Flick/Controllers/ListViewController.swift

```diff
@@ -193,3 +195,26 @@ extension ListViewController: UICollectionViewDelegateFlowLayout {
     }
 
 }
+
+extension ListViewController: MediaListHeaderDelegate, SortListModalDelegate {
+    func addMedia() {
+        print("add media")
+    }
+
+    func editMedia() {
+        print("edit media")
+    }
+
+    func sortMedia() {
+        sortListModalView = SortListModalView()
+        sortListModalView.delegate = self
+        navigationController?.navigationBar.layer.zPosition = -1;
```

Comment:
what does this do?

Reply:
Hm, this is actually not working 100% the way I want. but right now, when I show the popup, the the navigation bar does not get covered. so setting z index moves it to the back so it gets covered. but right now it ends up getting covered by the main view too.. :/